.devcontainer/devcontainer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "images",
+  "name": "Fedora",
```

Comment:
I don't use the devcontainer myself so I might be wrong about how this works, but doesn't this set the name for the container itself?  Is it a good idea to call it `Fedora` rather than something more specific, like the name of the project?

Reply:
One of the places image name is used - when GitHub codespaces created. Project name is evident without additional indication, but what what is codespace is about is not. That is why I think naming container is beneficial in this case. See screenshot.
<img width="834" alt="image" src="https://github.com/user-attachments/assets/fa1e6cc1-a831-4852-bc3f-6049702f8c65" />

Reply:
This isn't a major issue, btw.  Just curious about best practices.  If you'd rather keep it named "Fedora", I'm good to merge.

Reply:
I do not know best practices in regards to devcontainers unfortunately. Just relying on whatever seems reasonable.